src/helm/benchmark/metrics/image_generation/detectors/vitdet.py

```diff
@@ -0,0 +1,171 @@
+import os
+from typing import Dict, Any
+
+from PIL import Image
+import torch
+
+from helm.common.general import ensure_file_downloaded, hlog, get_helm_cache_path
+from helm.common.images_utils import open_image
+from helm.common.gpu_utils import get_torch_device
+from .base_detector import BaseDetector
+
+
+MODEL_CONFIG_DOWNLOAD_URL: str = "https://drive.google.com/uc?id=1MLuwQ0ZN0gJQ42oVCc0aFz6Rneb1g3Rt"
+MODEL_CHECKPOINT_DOWNLOAD_URL: str = (
+    "https://dl.fbaipublicfiles.com/detectron2/ViTDet/COCO/mask_rcnn_vitdet_b/f325346929/model_final_61ccd1.pkl"
+)
+
+
+class ViTDetDetector(BaseDetector):
+    def __init__(self):
+        from detectron2.checkpoint import DetectionCheckpointer
+        from detectron2.config import LazyConfig
+        from detectron2.config import instantiate
+        from detectron2.data.catalog import MetadataCatalog
```

Comment:
For all optional dependencies, could you do:

```python
try:
    from detectron2.checkpoint import DetectionCheckpointer
    from detectron2.config import LazyConfig
    from detectron2.config import instantiate
    from detectron2.data.catalog import MetadataCatalog
except ModuleNotFoundError as e:
    handle_module_not_found_error(e, ["heim"])
```

The `["heim"]` part requires #1944 to be merged first though.

Here's an [example](https://github.com/stanford-crfm/helm/blob/50e656526c9946ad3bc2cf32be5a52656ac7deb6/src/helm/benchmark/metrics/summarization_metrics.py#L57-L60).